docs/getting-started-guides/windows/index.md

```diff
@@ -55,19 +56,19 @@ There are several supported network configurations with Kubernetes v1.9 on Windo
 1. [Upstream L3 Routing](#upstream-l3-routing-topology) - IP routes configured in upstream ToR
 2. [Host-Gateway](#host-gateway-topology) - IP routes configured on each host
 3. [Open vSwitch (OVS) & Open Virtual Network (OVN) with Overlay](#overlay-using-ovn-controller-and-ovs-switch-extension-topology) - overlay networks (supports STT and Geneve tunneling types)
```

Comment:
Can you please also replace 
```
#overlay-using-ovn-controller-and-ovs-switch-extension-topology
```
with
```
#using-ovn-with-ovs
```
Thanks!

Reply:
yup. just did.